### PR TITLE
Change Attacker touches robot in defense penalties

### PR DESCRIPTION
**Pull request by rhololkeolke**
Remove stoppage and free kick.

Goals within 2 seconds of end of penalty do not count. Foul counter increases.

Rephrased Minor Offenses section as the old text is no longer accurate.

See https://docs.google.com/spreadsheets/d/1LfNNrE4ld0IkRGXSkKSKOnPDipdKbuH-InZJY2kATs4/edit?usp=sharing

### CODE REVIEW

**Comment by tobiasheineken** [comment]
I dislike that in this change (and similar ones) the games keeps running and goals are not awarded, what really makes it hard to follow the game. Also, seconds where the clock counts down while it's in the best interest of at least one teams to not let the game progress seem bad for the game.

**Comment by g3force** [comment]
I find the game harder to follow, when it is stopped and I also think that such a counter is reasonable or at least I do not know of any better alternative. 
I also don't think it is bad for the game, if a team can not score a goal. 10s is not that much time, so it can either just keep going or just try to keep ball control, which can also be interesting.

We will probably add the count down to the status board as well.

**Comment by MathewMacDougall** [comment]
I'm fine with this change because it already happens pretty infrequently, and we don't really know how these periods of time where goals are not allowed will affect the spectator experience. Personally I'm willing to try it.

**Comment by rhololkeolke** [comment]
Closed in favor of #32
